mappers/bluetooth_mapper/scheduler/scheduler.go

```diff
@@ -95,8 +95,8 @@ func (schedule *Schedule) performScheduleOperation(actionManager []actionmanager
 			schedule.Interval = defaultEventFrequency
 		}
 		if !actionExists {
-			klog.Errorf("Action %s does not exist. Exiting from schedule !!!", actionName)
-			break
+			klog.Errorf("Action %s does not exist.", actionName)
+			continue
```

Comment:
Thanks for the fixing, and what's the usage of `continue` here? No matter the actionExists  is true or false, the for will continue

Reply:
Yes, for there's a sleep after each schedule. When there's no action, I thought the sleep is not needed.
BTW, move this schedule.Interval setting out of this for rotation.